docs/scripts/model_building.rst

```diff
@@ -57,44 +58,86 @@ and some event characteristics (the features) to reconstruct the energy.
+  GridSearchCV:
+    use: False # True or False
+    # if False the following two variables are irrelevant
+    scoring: 'explained_variance'
+    cv: 2
 
   Method:
-   name: 'AdaBoostRegressor'
-   target_name: 'true_energy'
-   tuned_parameters:
-    learning_rate: [0.3]
-    n_estimators: [100]
-    base_estimator__max_depth: [null]  # null is equivalent to None
-    base_estimator__min_samples_split: [2]
-    base_estimator__min_samples_leaf: [10]
-   scoring: 'explained_variance'
-   cv: 2
-
+    name: 'sklearn.ensemble.AdaBoostRegressor'
+    target_name: 'true_energy'
+    # Please, see scikit-learn's API for what each parameter means
+    # NOTE: null == None
+    base_estimator:
+      name: 'sklearn.tree.DecisionTreeRegressor'
+      parameters:
+        # NOTE: here we set the parameters relevant for sklearn.tree.DecisionTreeRegressor
+        criterion: "mse" # "mse", "friedman_mse", "mae" or "poisson"
+        splitter: "best" # "best" or "random"
+        max_depth: null # null or integer
+        min_samples_split: 2 # integer or float
+        min_samples_leaf: 1 # int or float
+        min_weight_fraction_leaf: 0.0 # float
+        max_features: null # null, "auto", "sqrt", "log2", int or float
+        max_leaf_nodes: null # null or integer
+        min_impurity_decrease: 0.0 # float
+        random_state: 0 # null or integer or RandomState
+        ccp_alpha: 0.0 # non-negative float
+    tuned_parameters:
+      n_estimators: 50
+      learning_rate: 1
+      loss: 'linear' # 'linear', 'square' or 'exponential'
+      random_state: 0 # int, RandomState instance or None
+
+  # List of the features to use to train the model
+  # You can:
+  # - comment/uncomment the ones you see here,
+  # - add new ones here if they can be evaluated with pandas.DataFrame.eval
+  # - if not you can propose modifications to protopipe.mva.utils.prepare_data
   FeatureList:
-   - 'log10_hillas_intensity'
-   - 'log10_impact_dist'
-   - 'hillas_width_reco'
-   - 'hillas_length_reco'
-   - 'h_max'
-
+    Basic: # single-named, they need to correspond to input data columns
+    - 'h_max'         # Height of shower maximum from stereoscopic reconstruction
+    - 'impact_dist'   # Impact parameter from stereoscopic reconstruction
+    - 'hillas_width'  # Image Width
+    - 'hillas_length' # Image Length
+    # - 'concentration_pixel' # Percentage of photo-electrons in the brightest pixel
+    - 'leakage_intensity_width_1_reco' # fraction of total Intensity which is contained in the outermost pixels of the camera
+    Derived: # custom evaluations of basic features that will be added to the data
+      # column name : expression to evaluate using basic column names
+      log10_WLS: log10(hillas_width*hillas_length/hillas_intensity)
+      log10_intensity: log10(hillas_intensity)
+      CTAMARS_1: (sqrt((hillas_x - az)**2 + (hillas_y - alt)**2))**2
+      CTAMARS_2: arctan2(hillas_y - alt, hillas_x - az)
```

Comment:
I still think using human readable names for these is important. CTAMARS1 and CTAMARS2 gives no information about what they are.

Reply:
Yes sorry I forgot to change them